src/Autosuggest.js

```diff
@@ -74,6 +74,12 @@ class Autosuggest extends Component {
     super();
 
     this.saveInput = this.saveInput.bind(this);
+    this.onDocumentMouseUp = this.onDocumentMouseUp.bind(this);
+  }
+
+  componentDidMount() {
+    global.window.addEventListener('mouseup', this.onDocumentMouseUp, false);
+    global.window.addEventListener('touchend', this.onDocumentMouseUp, false);
```

Comment:
The name `onDocumentMouseUp` implies that it is attached to the `document`, not the `window`. Will it still do the trick if we attach it to the `document`?